pkgs/top-level/all-packages.nix

```diff
@@ -7707,6 +7707,7 @@ with pkgs;
     rubyBindings = config.radare.rubyBindings or false;
     luaBindings = config.radare.luaBindings or false;
   };
+  r2-cutter = qt5.callPackage ../development/tools/analysis/cutter { };
```

Comment:
Please use `libsForQt5`.

AUR names it `radare2-cutter` (https://aur.archlinux.org/packages/radare2-cutter/), would you like that name? I'd also rename the directory to `radare2-cutter` for consistency.

Reply:
Aye aye.